scripts/populate_preprint_providers.py

```diff
@@ -360,14 +361,666 @@ def main():
                 (['Law'], True),
                 (['Social and Behavioral Sciences'], True),
             ],
-        },
+        }
     ]
 
+    if env == 'stage':
+        STAGING_PREPRINT_PROVIDERS = [
```

Comment:
Because of the size of the preprint provider data, it might be better to just have a dictionary of all possible preprint providers, then make a list of all of the ids of the preprint providers we want on any given box. Then, when the script is run, it can just build the lists of dictionaries based on the list of ids. Error if any of the ids don't match.